src/main/java/com/palantir/gradle/versions/GradleComparators.java

```diff
@@ -27,20 +27,17 @@
 public final class GradleComparators {
     private GradleComparators() {}
 
-    public static final Comparator<ModuleIdentifier> MODULE_IDENTIFIER_COMPARATOR = Comparator
-            .comparing(ModuleIdentifier::getGroup)
-            .thenComparing(ModuleIdentifier::getName);
+    public static final Comparator<ModuleIdentifier> MODULE_IDENTIFIER_COMPARATOR =
+            Comparator.comparing(ModuleIdentifier::getGroup).thenComparing(ModuleIdentifier::getName);
 
     /**
-     * Compare {@link ModuleComponentIdentifier} using {@link #MODULE_IDENTIFIER_COMPARATOR}, but all other
-     * identifiers using {@link ComponentIdentifier#getDisplayName()}.
+     * Compare {@link ModuleComponentIdentifier} using {@link #MODULE_IDENTIFIER_COMPARATOR}, but all other identifiers
+     * using {@link ComponentIdentifier#getDisplayName()}.
      */
-    public static final Comparator<ComponentIdentifier> COMPONENT_IDENTIFIER_COMPARATOR = Comparator
-            .comparing(
-                    (ComponentIdentifier id) -> tryCast(ModuleComponentIdentifier.class, id),
-                    Comparators.emptiesFirst(Ordering
-                            .from(MODULE_IDENTIFIER_COMPARATOR)
-                            .onResultOf(ModuleComponentIdentifier::getModuleIdentifier)))
+    public static final Comparator<ComponentIdentifier> COMPONENT_IDENTIFIER_COMPARATOR = Comparator.comparing(
+                    (ComponentIdentifier id) -> tryCast(ModuleComponentIdentifier.class, id), Comparators.emptiesFirst(
+                                    Ordering.from(MODULE_IDENTIFIER_COMPARATOR)
```

Comment:
I don't think this would pass code-review if a human produced it - maybe we'd need to extract out to a variable so that the formatter will leave it in a good state?

(easier reading here: https://github.com/palantir/gradle-consistent-versions/blob/db658f7babf946e5f85578586756311448821ce3/src/main/java/com/palantir/gradle/versions/GradleComparators.java)

Reply:
Seems to have been fixed in the latest version: https://github.com/palantir/gradle-consistent-versions/blob/565fd9f3272a7ca7ba8f23d1e1ab22bd42f77911/src/main/java/com/palantir/gradle/versions/GradleComparators.java#L37-L41